src/main/scala/gov/uk/justice/digital/pollpush/services/NomisSource.scala

```diff
@@ -51,7 +51,7 @@ class NomisSource @Inject() (@Named("sourceUrl") sourceUrl: String, @Named("note
 
         case HttpResponse(_, _, entity, _) =>
 
-          Unmarshal(entity).to[Seq[SourceCaseNote]].map(PullResult(_, Some(from), Some(until), None))
+          Unmarshal(entity.withoutSizeLimit()).to[Seq[SourceCaseNote]].map(PullResult(_, Some(from), Some(until), None))
```

Comment:
There is no environment variable that can be used but we could use application.conf and add an arbitrary maximum; however I think this works better - we just want to grab whatever is available - if we have not got enough memory - tough luck we will die - but that is no worse than an exception.